src/shared/types/common.ts

```diff
@@ -129,6 +130,7 @@ export interface DLUser extends DLUserAccount {
     isFederationUser?: boolean;
     isGlobalFederationUser?: boolean;
     isLocalFederationUser?: boolean;
+    isFederationUserOnCommunity?: boolean;
```

Comment:
`withNavigation`

Reply:
added HIDE_NAVIGATION, by default will be false